src/org/javarosa/xform/parse/XFormParser.java

```diff
@@ -1713,8 +1712,15 @@ protected DataBinding processStandardBindAttributes(List<String> usedAtts, Eleme
             "saveIncomplete"
     ));
 
+    // Attributes that are passed through to additionalAttrs but shouldn't lead to warnings.
```

Comment:
I’m a big fan of having comments like these be doc comments, so that I can press F1 on the identifier where it is referenced, and see a helpful display like this:

![screen shot 2018-03-18 at 1 02 04 pm](https://user-images.githubusercontent.com/55099/37570426-e2fc9f74-2aac-11e8-8b67-578611eb1097.png)